<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Stitcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources_rc.qrc">
    <normaloff>:/statics/icon.ico</normaloff>:/statics/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="v_main_layout" stretch="1,0,10,0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="h_info_layout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="image_mode_lb">
            <property name="text">
             <string>接合模式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="horizon_rtn">
            <property name="toolTip">
             <string>圖片讀取時只載入檔名</string>
            </property>
            <property name="text">
             <string>水平</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">stitch_mode</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="vertical_rtn">
            <property name="toolTip">
             <string>圖片讀取時載入縮圖</string>
            </property>
            <property name="text">
             <string>垂直</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">stitch_mode</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="max_width_lb">
            <property name="text">
             <string>最大寬度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="max_width_value"/>
          </item>
          <item>
           <widget class="QLabel" name="max_heigth_lb">
            <property name="text">
             <string>最大高度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="max_heigth_value">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="items_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="movement">
         <enum>QListView::Free</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="item_list_info_lb">
          <property name="text">
           <string>雙擊列表物件可顯示放大圖片，滑鼠可拖曳物件來排序</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clear_list_btn">
          <property name="toolTip">
           <string>清空列表的物件</string>
          </property>
          <property name="text">
           <string>清空列表</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_rc.qrc">
            <normaloff>:/statics/clear_all.svg</normaloff>:/statics/clear_all.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="h_button_layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="toolTip">
           <string>開啟報告匯出設定視窗</string>
          </property>
          <property name="text">
           <string>開始</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_rc.qrc">
            <normaloff>:/statics/play_arrow.svg</normaloff>:/statics/play_arrow.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_btn">
          <property name="toolTip">
           <string>關閉應用程式</string>
          </property>
          <property name="text">
           <string>離開</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_rc.qrc">
            <normaloff>:/statics/logout.svg</normaloff>:/statics/logout.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>說明</string>
    </property>
    <addaction name="menu_about_manual"/>
    <addaction name="menu_about_update"/>
    <addaction name="menu_about_about"/>
   </widget>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_about_about">
   <property name="text">
    <string>關於</string>
   </property>
  </action>
  <action name="menu_about_manual">
   <property name="text">
    <string>使用說明</string>
   </property>
  </action>
  <action name="menu_about_update">
   <property name="text">
    <string>檢查更新</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources_rc.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="stitch_mode"/>
 </buttongroups>
</ui>
